cmd/kops/create_cluster.go

```diff
@@ -80,6 +81,8 @@ func init() {
 	cmd.Flags().StringVar(&createCluster.DNSZone, "dns-zone", "", "DNS hosted zone to use (defaults to last two components of cluster name)")
 	cmd.Flags().StringVar(&createCluster.OutDir, "out", "", "Path to write any local output")
 	cmd.Flags().StringVar(&createCluster.AdminAccess, "admin-access", "", "Restrict access to admin endpoints (SSH, HTTPS) to this CIDR.  If not set, access will not be restricted by IP.")
+
+	cmd.Flags().BoolVar(&createCluster.NoAssociatePublicIP, "no-associate-public-ip", false, "Specify --no-associate-public-ip to disable association of public IP for master ASG and nodes.")
```

Comment:
Just wondering why you went with "NoAssociatePublicIP" instead of "AssociatePublicIP".

If it was because you didn't want it to show up in `kops edit instancegroup`, I don't think it's a problem.

But maybe it was because you wanted it to default to true in the template?  That is interesting and might well require a helper function...